includes/RemoteWiki.php

```diff
@@ -46,7 +46,8 @@ protected static function newFromConds(
 	}
 
 	private function determineCreationDate() {
-		$res = wfGetDB( DB_MASTER )->selectField(
+		global $wgCreateWikiGlobalWiki;
+		$res = wfGetDB( DB_MASTER, [], $wgCreateWikiGlobalWiki )->selectField(
```

Comment:
minor but this should be DB_REPLICA now anyway :)

Reply:
Done.